frontend/common/lang.js

```diff
@@ -22,8 +22,11 @@ i18next.use(LanguageDetector).init({
         el: {
             translation: ellinika,
         },
+        de: {
+            translation: without_empty_keys(deutsch),
+        },
```

Comment:
Also this whole block might be nicer ordered alphabetically, especially when we later have more languages.